modules/nf-core/phispy/tests/main.nf.test

```diff
@@ -0,0 +1,67 @@
+// nf-core modules test phispy
+nextflow_process {
+
+    name "Test Process PHISPY"
+    script "../main.nf"
+    process "PHISPY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "phispy"
+
+    test("bacteroides_fragilis - gbff") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['bacteroides_fragilis']['genome']['genome_gbff_gz'], checkIfExists: true)
```

Comment:
This should be changed to use the new param `params.modules_testdata_base_path`, e.g.,
```
file( params.modules_testdata_base_path + "/genomics/prokaryotes/bacteroides_fragilis/illumina/fastq/test1_1.fastq.gz", checkIfExists: true )
```

Reply:
Done in [eb66705](https://github.com/nf-core/modules/pull/5292/commits/eb66705587db232fc45ad84a6f34549270d16ca6)